pep-0999.rst

```diff
@@ -0,0 +1,175 @@
+PEP: 999
+Title: Final Grammar for Literal String Interpolation
+Version: $Revision$
+Last-Modified: $Date$
+Author: Philipp Angerer <phil.angerer@gmail.com>
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 20-Sep-2016
+Python-Version: 3.7
+Post-History: 20-Sep-2016
+Resolution: None
```

Comment:
There is no point to keep an optional field now (and I don't think using None as a value makes any sense) We can always add it back when your PEP is discussed on python-ideas :)